docs/BUILD.md

```diff
@@ -674,3 +674,110 @@ The latter aspect makes maintaining `lfedge/eve-alpine` a bit tricky, even thoug
 Step #2 guarantees that all _existing_ packages will simply be re-used from the previous version of the cache and _NOT_ re-downloaded from the Alpine http mirrors (remember that re-downloading always runs the risk of getting a different version of the same package). Step #1, of course, will download new packages and pin them in the new version of the cache.
 
 If the above seems a bit confusing, don't despair: here's an [actual example](../../../commit/1340c1b6981ed3219f78a7a0209ff9222a0dacdf) of adding 4 new packages `libfdt`, `dtc`, `dtc-dev` and `uboot-tools` to the `lfedge/eve-alpine` cache.
+
+## Cross-compilation support
+
+### cross-compilers package
+
+In order to enable cross-compilation we add `pkg/cross-compilers` which contains `binutils`, `gcc` and `g++` prepared with target architecture different from host one.
+We support `aarch64` and `riscv64` target architectures for `x86_64` host architecture and `x86_64` and `riscv64` target architectures for `aarch64` host architecture.
```

Comment:
I put the table here.

> Also, is this just for kernel, or other packages as well? And for other packages, do we need the cross-compilers package if we are doing go? Or is it just if we use CGO?

It is about the whole set of packages, I did not tell about specific package here. In case of CGO, we need gcc with cross-compile support, and we should see into the table to understand, do we have support for our build-target pair.